compiler/Bond/Template/CustomMapping.hs

```diff
@@ -42,9 +50,9 @@ integer = decimal <$> many1 digit <?> "decimal number"
 -- parse alias mapping specification from the command line --using flags
 -- e.g.: --using="OrderedSet=SortedSet<{0}>"
 parseAliasMapping :: [String] -> IO [AliasMapping]
-parseAliasMapping = mapM parseAliasMapping
+parseAliasMapping = mapM parseAliasMapping'
```

Comment:
Maybe better names would be:

    parseAliasMappings = mapM parseAliasMapping

?